### DISCUSSION TITLE
Tauri has requirements for opengl version, can opengl 3.3 work?

### DISCUSSION THREAD

**Discussion by xiaguangbo** [tauri-apps/tauri#9785] [Q&A]
I tested it on a RISCV JH7110 development board, Start from the desktop graphics command line：
```
Could not create EGL sharing context
Could not create EGL context
```

Use `glxinfo | grep "OpenGL version"` to get version information
JH7110 development board is opengl 3.3 mesa 22

**Comment by FabianLars** [comment]
I couldn't really find anything specific about webkitgtk's OpenGL requirements but just to double check, can you test it with the `WEBKIT_DISABLE_COMPOSITING_MODE=1` and `WEBKIT_DISABLE_DMABUF_RENDERER=1` env vars set?

**Reply by xiaguangbo** [reply]
no effect. `glxgears -info` can see the gears. `export RUST_BACKTRACE=1` not see the effect.
libwebkit4.0、4.1、5.0 all tested.
I may need to manually compile webkit to try

**Reply by xiaguangbo** [reply]
/usr/lib/riscv64-linux-gnu/webkit2gtk-5.0/MiniBrowser:
```
Could not create EGL sharing context
Could not create EGL context
Segmentation fault
```

**Reply by xiaguangbo** [reply]
4.0 4.1 although this error will be reported, but can open the browser.
/usr/lib/riscv64-linux-gnu/webkit2gtk-4.0/MiniBrowser
/usr/lib/riscv64-linux-gnu/webkit2gtk-4.1/MiniBrowser
```
Could not create EGL sharing context
Could not create EGL context
```
